command/output/cli_output.go

```diff
@@ -0,0 +1,32 @@
+package output
```

Comment:
Do you think we can get rid of the output package and move all of it's contents to `/command/output.go`?

Reply:
Moved 👍 

[eecd37b](https://github.com/0xPolygon/polygon-edge/pull/410/commits/eecd37b09a508d36ea401de9d9e69cc4c4d4dc31)